queries/haskell_persistent/highlights.scm

```diff
@@ -0,0 +1,38 @@
+;; ----------------------------------------------------------------------------
+;; Literals and comments
+
+(integer) @number
+(float) @float
+(char) @character
+(string) @string
+(attribute_name) @attribute
+(attribute_exclamation_mark) @attribute
+
+(con_unit) @symbol  ; unit, as in ()
+
+(comment) @comment @spell
+
+;; ----------------------------------------------------------------------------
+;; Keywords, operators, includes
+
+[
+  "Id"
+  "Primary"
+  "Foreign"
+  "deriving"
+] @keyword
+
+"=" @punctuation.delimiter
```

Comment:
does `@operator` fit better?

Reply:
Yes, in HTML and Haskell syntaxes it is an @operator.